bin/install-wp-suite.sh

```diff
@@ -69,6 +73,13 @@ installWpSuite() {
   - WP version: ${INFO_C}${WP_VERSION}${NO_C}
   - Skip DB creation: ${INFO_C}${SKIP_DB_CREATION}${NO_C}"
 	. "$PARENT_DIR/install-wp-tests.sh" "$DB_NAME" "$DB_USER" "$DB_PASS" "$DB_HOST" "$WP_VERSION" "$SKIP_DB_CREATION"
+
+	# Shut up.
+	set +x
+
+	if [[ ! -n "$3" || 'true' == $3 ]]; then
+		clear_patchwork_cache
+	fi
```

Comment:
```suggestion
	clear_patchwork_cache
```
Let's do that any time we install WP.